features.md

```diff
@@ -0,0 +1,335 @@
+  The value MUST NOT be less than the value of the `ociVersionMin` property.
+  The feature document MUST NOT contain properties that are not defined in this version of the Open Container Initiative Runtime Specification.
+
+### Example
+```json
+{
+  "ociVersionMin": "1.0.0",
+  "ociVersionMax": "1.1.0"
+}
+```
+
+## <a name="featuresHooks" />Hooks
+* **`hooks`** (array of strings, OPTIONAL) The recognized names of the [hooks](config.md#hooks).
+  The runtime MUST support the elements in this array as the [`hooks` property of `config.json`](config.md#hooks).
+
+### Example
+```json
+"hooks": [
+  "prestart",
+  "createRuntime",
+  "createContainer",
+  "startContainer",
+  "poststart",
+  "poststop"
+]
+```
+
+## <a name="featuresMountOptions" />Mount Options
+
+* **`mountOptions`** (array of strings, OPTIONAL) The recognized names of the mount options, including options that might not be supported by the host operating system.
+  The runtime MUST recognize the elements in this array as the [`options` of `mounts` objects in `config.json`](config.md#mounts).
+  * Linux: this array SHOULD NOT contain filesystem-specific mount options that are passed to the [mount(2)][mount.2] syscall as `const void *data`.
+
+### Example
+
+```json
+"mountOptions": [
+  "acl",
+  "async",
+  "atime",
+  "bind",
+  "defaults",
+  "dev",
+  "diratime",
+  "dirsync",
+  "exec",
+  "iversion",
+  "lazytime",
+  "loud",
+  "mand",
+  "noacl",
+  "noatime",
+  "nodev",
+  "nodiratime",
+  "noexec",
+  "noiversion",
+  "nolazytime",
+  "nomand",
+  "norelatime",
+  "nostrictatime",
+  "nosuid",
+  "nosymfollow",
+  "private",
+  "ratime",
+  "rbind",
+  "rdev",
+  "rdiratime",
+  "relatime",
+  "remount",
+  "rexec",
+  "rnoatime",
+  "rnodev",
+  "rnodiratime",
+  "rnoexec",
+  "rnorelatime",
+  "rnostrictatime",
+  "rnosuid",
+  "rnosymfollow",
+  "ro",
+  "rprivate",
+  "rrelatime",
+  "rro",
+  "rrw",
+  "rshared",
+  "rslave",
+  "rstrictatime",
+  "rsuid",
+  "rsymfollow",
+  "runbindable",
+  "rw",
+  "shared",
+  "silent",
+  "slave",
+  "strictatime",
+  "suid",
+  "symfollow",
+  "sync",
+  "tmpcopyup",
+  "unbindable"
+]
+```
+
+
+## <a name="featuresPlatformSpecificFeatures" />Platform-specific features
+
+* **`linux`** (object, OPTIONAL) [Linux-specific features](features-linux.md).
+  This MAY be set if the runtime supports `linux` platform.
+
+## <a name="featuresAnnotations" />Annotations
+
+**`annotations`** (object, OPTIONAL) contains arbitrary metadata of the runtime.
+This information MAY be structured or unstructured.
+Annotations MUST be a key-value map that follows the same convention as the Key and Values of the [`annotations` property of `config.json`](config.md#annotations).
+
+### Example
+```json
+"annotations": {
+  "org.opencontainers.runc.checkpoint.enabled": "true",
+  "org.opencontainers.runc.version": "1.1.0"
+}
+```
+
+# Example
+
+Here is a full example for reference.
```

Comment:
(likely to go stale before even getting a new release of this spec out) :-)

Reply:
Which property going stale?